src/pipeline.ml

```diff
@@ -191,7 +194,7 @@ module Tarides = struct
               [];
           ]
           ~archs:[`Linux_x86_64; `Linux_arm64; `Linux_ppc64; `Linux_s390x; `Linux_riscv64]
-          ~options:(include_git |> fun v ->  { v with build_args =  ["--ulimit stack=1000000000:1000000000"]});
+          ~options:(defaults |> include_git |> fun v ->  { v with build_args =  ["--ulimit stack=1000000000:1000000000"]});
```

Comment:
Just a stylistics note: I'm not convinced that these record update functions improve readability over OCaml's build in record update syntax. Using the latter, this would be

```suggestion
          ~options:({ defaults with include_git = true; build_args =  ["--ulimit stack=1000000000:1000000000"]});
```

The other instances would looks similar, and anyone familiar with OCaml would know what was happening without having to look up the definition of `include_git` and `build_kit`.